frontend/src/views/ExtractView.vue

```diff
@@ -82,6 +82,13 @@ const extractFrame = async () => {
     selectedFrame.value = framesList.value.indexOf(data.value[0]);
   }
 };
+
+// Change player to frame of clicked of element
+const clickFrame = frameName => {
```

Comment:
You should provide the type for `frameName` here, vscode actually has a quick fix here to automatically infer type params from its usages.
```suggestion
const clickFrame = (frameName: string) => {
```